tests/view/ViewableDataTest.php

```diff
@@ -253,9 +269,14 @@ public function setValue() {}
 class ViewableDataTest_Container extends ViewableData {
 
 	public function __construct($failover) {
-		$this->failover = $failover;
+		$this->setFailover($failover);
 		parent::__construct();
 	}
+
+	public function setFailover($failover) {
```

Comment:
Ah yes, I only added that to the test class. I think I’ll redo this PR and add it to `ViewableData`, we should probably have a setter anyway.